packages/engine/src/simulation/package/context/packages/neighbors/mod.rs

```diff
@@ -162,4 +169,8 @@ impl Package for Neighbors {
 
         Ok(vec![(field_key, Arc::new(neighbors_builder.finish()))])
     }
+
+    fn get_span(&self) -> Span {
```

Comment:
In most cases, getters in Rust omit `get_`.
```suggestion
    fn span(&self) -> Span {
```